rom/merge.cpp

```diff
@@ -146,14 +146,15 @@ void LoadSampleSets(const int rank, const double energyFraction, const int nsets
     }
     std::string basis_filename = basename + "/basis" + varName + std::to_string(basisWindow) + basisIdentifier;
 
+    int first_sv = (useOffset && offsetType == useInitialState && basisWindow > 0) && (v == X || v == V || v == E);;
```

Comment:
extra `;`

Reply:
Thanks for pointing that out! I just removed the extra one.